### ISSUE TITLE
User account

### ISSUE THREAD

**Issue by uihdff** [blackfuel/rtlentropy-arm-asuswrt#1]
Is it possible to code this application to use a non-Admin account ?

**Comment by blackfuel** [comment]
Yes, I should probably move the 'sudo' commands to a separate script.  It's for configuring the AsusWRT build environment.

**Comment by blackfuel** [comment]
My scripts are for cross-compiling other peoples' work to run on AsusWRT routers.  The original project is here:  https://github.com/pwarren/rtl-entropy.git

I wasn't sure if your question was about cross-compiling as non-admin user or running rtl-entropy on the router as non-admin user.  I agree, it would be better security to run rtl-entropy on the router as non-admin user.
